src/test/ui/raw-ident-suggestion.rs

```diff
@@ -0,0 +1,22 @@
+#![allow(non_camel_case_types)]
```

Comment:
Let's move this file to `ui/issues/issue-65634-raw-ident-suggestion.rs` to avoid cluttering `ui/`